generated/nidcpower/nidcpower/lcr_measurement.py

```diff
@@ -0,0 +1,229 @@
+        ("ac_voltage_real"     , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("ac_voltage_imaginary", nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("ac_current_real"     , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("ac_current_imaginary", nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("z_real"              , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("z_imaginary"         , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("z_magnitude"         , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("z_phase"             , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("y_real"              , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("y_imaginary"         , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("y_magnitude"         , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("y_phase"             , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("ls"                  , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("cs"                  , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("rs"                  , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("lp"                  , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("cp"                  , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("rp"                  , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("d"                   , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("q"                   , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("measurement_mode"    , nidcpower._visatype.ViUInt16 ),  # noqa: E202,E203
+        ("dc_in_compliance"    , nidcpower._visatype.ViBoolean),  # noqa: E202,E203
+        ("ac_in_compliance"    , nidcpower._visatype.ViBoolean),  # noqa: E202,E203
+        ("unbalanced"          , nidcpower._visatype.ViBoolean),  # noqa: E202,E203
+        ("reserved1"           , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("reserved2"           , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("reserved3"           , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("reserved4"           , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("reserved5"           , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("reserved6"           , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("reserved7"           , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+    ]
+
+
+class LCRMeasurement(object):
+    """Specifies an LCR measurement.
+
+    Fields:
+        channel (str): The channel name associated with this LCRMeasurement.
+
+        vdc (float): Specifies the measured DC voltage, in volts.
+
+        idc (float): Specifies the measured DC current, in amps.
```

Comment:
amperes

Reply:
`amps` is actually the convention used across the API documentations for this custom struct, e.g. in LabVIEW:
![image](https://user-images.githubusercontent.com/80089084/179161108-7d39b475-543c-457d-9170-a65fc63ef2b0.png)

In fact, the documentations for the fields in LCRMeasurement are largely based on the corresponding LabVIEW documentations for these fields if applicable. For consistency, I believe we should stick to the same wordings across different APIs whenever possible.